qemu/tests/cfg/rebase_onto_no_backing_file.cfg

```diff
@@ -8,18 +8,16 @@
     # md5sum binary path
     md5sum_bin = "md5sum"
     image_chain = "image1 sn"
-    image_format_image1 = "raw"
     image_name_sn = "images/sn"
     # set size to "", so during snapshot creation
     # the cmdline will not have specified size option
     image_size_sn = ""
     image_format_sn = qcow2
-    tmp_dir = /var/tmp
-    tmp_file_name = ${tmp_dir}/testfile
-    file_create_cmd = "dd if=/dev/urandom of=%s bs=1M count=512"
-    guest_file_name = ${tmp_file_name}
+    guest_tmp_filename = "/tmp/sn"
+    backup_image_before_testing = yes
+    restore_image_after_testing = yes
     Windows:
-        guest_file_name = C:\testfile
+        guest_tmp_filename = "C:\\sn"
```

Comment:
Changing `sn` to `base` would be better? For the file is written in the base file.